### PR TITLE
[10][MIGR] account invoice pricelist sale

### PR DESCRIPTION
**Pull request by rvalyi**
Depends on
- [x] account_invoice_pricelist: #368

### CODE REVIEW

**Comment by rvalyi** [comment]
#342 has been replaced by https://github.com/OCA/account-invoicing/pull/368 so it depends on #368 now

**Comment by rvalyi** [comment]
ping @bealdav @pedrobaeza 
the https://github.com/OCA/account-invoicing/pull/368 dependency has been merged. How is it for merging this one now?

**Comment by pedrobaeza** [comment]
Can you please rebase first?

File: account_invoice_pricelist_sale/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
```
Comment (bealdav) [inline account_invoice_pricelist_sale/models/sale_order.py]: replace buy odoo

Comment (rvalyi) [reply account_invoice_pricelist_sale/models/sale_order.py]: done

**Comment by rvalyi** [comment]
@pedrobaeza rebase done, Travis green.

**Comment by rvalyi** [comment]
ping @pedrobaeza, not sure you have seen the rebase.

**Review by pedrobaeza — APPROVED** [review]
Although you can still squash 2 "OCA Transbot" commits

**Comment by rvalyi** [comment]
@pedrobaeza squash done cc @bealdav

**Review by legalsylvain — APPROVED** [review]
Hi @rvalyi. The commit text is confusing. (V11 written). Otherwise, :+1: 
thanks !

**Review by bealdav — APPROVED** [review]

**Comment by rvalyi** [comment]
@legalsylvain commit messages fixed. cc @pedrobaeza

**Pull request merged**